v3/mqtt/mqtt.go

```diff
@@ -0,0 +1,49 @@
+package mqtt
+
+import (
+	"errors"
+
+	"github.com/spinframework/spin-go-sdk/v3/internal/fermyon/spin/v2.0.0/mqtt"
+	"go.bytecodealliance.org/cm"
+)
+
+// OpenConnection initializes an MQTT connection
+func OpenConnection(address, username, password string, keepAliveIntervalInSecs uint64) (mqtt.Connection, error) {
```

Comment:
I've been trying to prevent exposing generated types. Can you create a Connection struct to wrap the `mqtt.Connection`?